app/scripts/components/boilerplate/header.jsx

```diff
@@ -10,8 +10,9 @@ const Header = React.createClass({
     return (
       <div className="header">
         <div className="logo">
-          <h1><T k="site-name" /></h1>
+          <img src="images/tz-flag.png"/>
```

Comment:
All `<img>` tags should have an `alt` prop for accessibility. It's actually hard in this case because our `<T>` translation component only renders strings, not props.

@esoliani I made a component for setting translated props, just wondering if you think it's way to ugly and we should do something else. It works like this:

```js
<div className="logo">
  <TSetChildProp props={{alt: 'site.name'}}>
    <img src="/images/tz-flag.png" />
  </TSetChildProps>
</div>
```

so, that would inject `alt="Water Dashboards"` into the `<img>` tag.

Not sure a better way to do it that would still automatically update the translation when the language is changed.